server/db/models/order.js

```diff
@@ -0,0 +1,33 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Order = db.define('order', {
+  startLocationLat: {
+    type: Sequelize.DECIMAL(10, 6)
+  },
+  startLocationLng: {
+    type: Sequelize.DECIMAL(10, 6)
+  },
+  pickupTime: {
+    type: Sequelize.DATE
+  },
+  deliveryTime: {
+    type: Sequelize.DATE
+  },
+  price: {
+    type: Sequelize.DECIMAL(10, 2),
```

Comment:
@danieleimer To avoid issues with adding prices, we should use integers, and will be cents